api/v1beta1/common_openstackbaremetalset.go

```diff
@@ -144,12 +162,15 @@ func VerifyBaremetalSetScaleUp(log logr.Logger, instance *OpenStackBaremetalSet,
 
 		// If we can't satisfy the new requested replica count, explicitly state so
 		if newBmhsNeededCount > len(availableBaremetalHosts) {
-			return nil, fmt.Errorf("Unable to find %d requested BaremetalHost count (%d in use, %d available)",
+			return nil, fmt.Errorf("Unable to find %d requested BaremetalHost count (%d in use, %d available)%s for OsBaremetalSet %s",
```

Comment:
I would use OpenstackBaremetalSet vs OsBaremetalSet to avoid confusion (or just use the Kind here)

Reply:
I was maintaining parity with other places where I had apparently used `OsBaremetalSet` (such as L105), but yeah, it would be better to just spell it out or use Kind.